tests/ci/integration/python_patch/main/aws-lc-cpython.patch

```diff
@@ -0,0 +1,555 @@
+diff --git a/Lib/test/test_asyncio/test_events.py b/Lib/test/test_asyncio/test_events.py
+index b25c097573..8f07868b82 100644
+--- a/Lib/test/test_asyncio/test_events.py
++++ b/Lib/test/test_asyncio/test_events.py
+@@ -1125,12 +1125,12 @@ def test_create_server_ssl_match_failed(self):
+         # incorrect server_hostname
+         f_c = self.loop.create_connection(MyProto, host, port,
+                                           ssl=sslcontext_client)
++        regex = "IP address mismatch, certificate is not valid for '127.0.0.1'"
++        if ssl is not None and "AWS-LC" in ssl.OPENSSL_VERSION:
++            regex = "CERTIFICATE_VERIFY_FAILED"
+         with mock.patch.object(self.loop, 'call_exception_handler'):
+             with test_utils.disable_logger():
+-                with self.assertRaisesRegex(
+-                        ssl.CertificateError,
+-                        "IP address mismatch, certificate is not valid for "
+-                        "'127.0.0.1'"):
++                with self.assertRaisesRegex(ssl.CertificateError, regex):
+                     self.loop.run_until_complete(f_c)
+ 
+         # close connection
+diff --git a/Lib/test/test_httplib.py b/Lib/test/test_httplib.py
+index 089bf5be40..1713a5fbae 100644
+--- a/Lib/test/test_httplib.py
++++ b/Lib/test/test_httplib.py
+@@ -2066,7 +2066,7 @@ def test_host_port(self):
+ 
+     def test_tls13_pha(self):
+         import ssl
+-        if not ssl.HAS_TLSv1_3:
++        if not ssl.HAS_TLSv1_3 or "AWS-LC" in ssl.OPENSSL_VERSION:
```

Comment:
Instead of checking for AWS-LC is it reasonable to add a new flag like ssl.HAS_PHA?

Reply:
We could propose that upstream, but I'm not sure whether it's worth adding a new public property to the `ssl` module API. Regarding this change, it would just streamline a few tests.

Reply:
Ahhh is this an external ssl module? I was thinking this was a part of some test config.

Reply:
Sorry, the term "module" is a bit [overloaded](https://docs.python.org/3/tutorial/modules.html). This `ssl` thing is CPython's standard library implementation of TLS, which is a thin wrapper around OpenSSL. So, whatever we properties we put on the `ssl` object become public APIs requiring stability, maintenance, etc.